torch/autograd/gradcheck.py

```diff
@@ -2130,14 +2134,16 @@ def gradgradcheck(
     fast_mode: bool = False,
     masked: bool = False,
 ) -> bool:
-    r"""Check gradients of gradients computed via small finite differences
+    r"""Verify gradient of gradients using :func:`~torch.allclose` for correct backpropagation.
+
+    Check gradients of gradients computed via small finite differences
```

Comment:
Does this change look good to you @albanD ?

Reply:
Not sure why the first sentence is repeated twice here and above?

Also I don't think we should mention allclose here. These functions are used to verify the gradients computed via autograd using finite difference as a reference.

Reply:
The idea is to have a short one-liner description and then a longer description according to pydocstyle. Can the one-liner be simply: "Verify gradient of gradients." and the longer one as is?

Reply:
I have made the changes, please review this.

Reply:
Yes sounds ok for the short description.